tests/src/test/java/io/streamnative/pulsar/handlers/kop/TransactionTest.java

```diff
@@ -898,7 +898,7 @@ public void testPurgeAbortedTx(boolean takeSnapshotBeforeRecovery) throws Except
 
         // unload and reload in order to have at least 2 ledgers in the
         // topic, this way we can drop the head ledger
-        admin.namespaces().unload(namespace);
+        admin.topics().unload(fullTopicName.getPartition(0).toString());
```

Comment:
the changes in this test demonstrate that now S4K is able to deal with single topic unload